### ISSUE TITLE
Lesson Question button contrast

### ISSUE THREAD

**Issue by msoupcoff** [DukeLearningInnovation/duke-sakai-skins#234]
In a Lesson, add a question. Enter lesson as student and select an answer. Submit. Button text fails color contrast.

Ex: [https://warpwire.duke.edu/w/wfEEAA/](https://warpwire.duke.edu/w/wfEEAA/)
